src/Kunstmaan/TranslatorBundle/Controller/TranslatorController.php

```diff
@@ -239,9 +276,12 @@ public function deleteAction(Request $request, $id)
             $em->getRepository('KunstmaanTranslatorBundle:Translation')->removeTranslations($id);
         }
 
-        return new RedirectResponse($this->generateUrl($indexUrl['path'], isset($indexUrl['params']) ? $indexUrl['params'] : array()));
+        return new RedirectResponse($this->generateUrl($indexUrl['path'], isset($indexUrl['params']) ? $indexUrl['params'] : []));
```

Comment:
This line is doing far too much. Can you set $params with the ternary on its own line ? Also $url. Then you can return new RedirectResponse($url, $params);